iot-hub/Samples/device/PnpDeviceSamples/TemperatureController/Program.cs

```diff
@@ -15,8 +15,10 @@ namespace Microsoft.Azure.Devices.Client.Samples
 {
     public class Program
     {
-        // DTDL interface used: https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/samples/TemperatureController.json
-        private const string ModelId = "dtmi:com:example:TemperatureController;1";
+        // DTDL interface used: https://github.com/Azure/iot-plugandplay-models
+        // in v2 of TempController Model, we use 2 different model of Thermostat, 
```

Comment:
This was confusing to read for me until I looked at the model again carefully. May be good to add information for components something like - TempController model has two components, one following v1 and the second v2 demonstrating that IoT central can handle both model versioning